### DISCUSSION TITLE
Is there a way I can have new pages automatically add links on a different page?

### DISCUSSION THREAD

**Discussion by kpkpkpkpkpkpkpkpkpkp** [requarks/wiki#6686] [Help / Questions] [answered]
Using the example on the wiki.js, say I have this pathing.

Page at path /universe with title "The Universe"
Page at path /universe/planets with title "The Planets"
Page at path /universe/planets/earth with title "The Blue Marble"

Is it possible to create a new page under the 'planets' path called 'mars', then have it add a new line under the existing 'planets' page that contains a link to new 'mars' page?

**Comment by NGPixel** [comment, marked as answer]
No, this is coming in v3.
